tests/integration/internal/logsgenerator/logsgenerator.go

```diff
@@ -0,0 +1,133 @@
+package logsgenerator
+
+import (
+	"fmt"
+	"strings"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+/// This package wraps the logsgenerator utility in sumologic-kubernetes-tools in a procedural API
+/// easily consumable by tests.
+
+const deploymentSleepTime = 60 * 60 * 24 // how much time we spend sleeping after generating logs in a Deployment
```

Comment:
Can we declare that as `time.Duration` with a time unit and then where needed convert to (seconds?) whatever is used here to prevent "implicit" time units usage.